pkg/uploader/provider/provider.go

```diff
@@ -42,3 +50,21 @@ type Provider interface {
 	// Close which will close related repository
 	Close(ctx context.Context)
 }
+
+func NewUploaderProvider(
+	ctx context.Context,
+	legacyUploader bool,
```

Comment:
Change it to uploader_type as complied with the value in PVB/PVR?

Reply:
yeah, I've changed